app/Models/Monster.php

```diff
@@ -246,6 +246,17 @@ public function setBase64ImageAttribute($value)
         $this->attributes['base64_image'] = $value;
     }
 
+    public function setDropzoneAttribute($value)
+    {
+        if (app('env') === 'production') {
+            \Alert::warning('In the online demo the base64 images don\'t get stored.');
+
+            return true;
+        }
+
+        $this->attributes['dropzone'] = $value;
+    }
+
```

Comment:
This is a nice catch. But isn't the upload actually done inside the Dropzone operation? This bit still wouldn't stop people from uploading malicious files to our demo server, right? Afaik we need to override the `dropzone()` method (or whatever it's called) in MonsterCrudController, to not let people upload files in the first place.

Reply:
Nice catch 👍 

Did the changes. 🙏